python/chronos/src/bigdl/chronos/data/utils/public_dataset.py

```diff
@@ -208,6 +211,26 @@ def preprocess_uci_electricity(self):
         self.df.value = self.df.value.apply(lambda x: str(x).replace(",", "")).astype(np.float32)
         return self
 
+    def preprocess_uci_electricity_wide(self):
+        '''
+        return data that meets the minimum requirements of tsdata.
+        '''
+        if not os.path.exists(self.final_file_path):
+            zip_file = zipfile.ZipFile(os.path.join(
+                                       os.path.expanduser(self.dir_path),
+                                       DATASET_NAME[self.name[:-5]][0]))
```

Comment:
why we need this `[:-5]`

Reply:
Oh, I forgot I already add 'uci_electricity_wide' in DATASET_NAME. "[:-5]" was added for if we don't modify DATASET_NAME.